.travis.yml

```diff
@@ -40,7 +40,7 @@ matrix:
 
 
 before_install:
-  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update;fi
+  - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew update && brew install libuv;else sudo -E apt-add-repository -y ppa:linuxjedi/ppa && sudo apt-get -qq update  && sudo apt-get install make automake libtool curl && curl -sSL https://github.com/libuv/libuv/archive/v1.8.0.tar.gz | sudo tar zxfv - -C /usr/local/src && TEMP=`pwd` && cd /usr/local/src/libuv-1.8.0 && sudo sh autogen.sh && sudo ./configure && sudo make && sudo make install && sudo rm -rf /usr/local/src/libuv-1.8.0 && cd $TEMP && sudo ldconfig;fi
```

Comment:
I want to have more complex test for this, inner flags etc. but I always find just some dev modules.